### PR TITLE
fix: try to use custom function name first

### PR DESCRIPTION
**Pull request by fengmk2**
pick from https://github.com/eggjs/egg/pull/2156

##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

### CODE REVIEW

**Comment by fengmk2** [comment]
@dead-horse 好了

File: test/fixtures/apps/ctx-background/app/controller/custom.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+const fs = require('mz/fs');
+
+module.exports = function* () {
```
Comment (fengmk2) [inline test/fixtures/apps/ctx-background/app/controller/custom.js]: 这里我改成 aa

File: test/fixtures/apps/ctx-background/app/controller/custom.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+const fs = require('mz/fs');
+
+module.exports = async (ctx) => {
```
Comment (dead-horse) [inline test/fixtures/apps/ctx-background/app/controller/custom.js]: async ctx =>

**Comment by fengmk2** [comment]
fixed

File: test/fixtures/apps/ctx-background/app/controller/custom.js (hunk 2 above)
Comment (popomore) [reply test/fixtures/apps/ctx-background/app/controller/custom.js]: 这个 lint 不会报？

Comment (dead-horse) [reply test/fixtures/apps/ctx-background/app/controller/custom.js]: fixtures ignore 了吧

**Pull request merged**